helm/values.yaml

```diff
@@ -0,0 +1,204 @@
+environment: prod
+
+image:
+  name: SET-BY-CICD-IMAGE
+  tag: SET-BY-CICD-TAG
+
+ingress:
+  enabled: false
+  host:
+  className:
+  tls:
+    enabled: false
+    secretName:
+
+redis:
+  enabled: true
+  architecture: standalone
+  fullnameOverride: montandon-etl-redis
+  auth:
+    enabled: false
+  master:
+    persistence:
+      enabled: true
+      size: .5Gi
+      storageClass:
+
+# https://artifacthub.io/packages/helm/bitnami/minio
+# extraEnvVars: https://github.com/bitnami/containers/blob/main/bitnami/minio/README.md#environment-variables
+minio:
+  enabled: false
+  disableWebUI: true
+  mode: standalone
+  fullnameOverride: montandon-etl-minio
+  global:
+    defaultStorageClass:
+  apiIngress:
+    enabled: true
+    ingressClassName:  # NOTE: Define me when used
+    hostname:  # NOTE: Define me when used
+    annotations:
+      nginx.ingress.kubernetes.io/proxy-body-size: "500m"
+  auth:
+    forceNewKeys: True
+    rootUser: montandon-etl
+    rootPassword:  # NOTE: Define me when used
+  persistence:
+    enabled: true
+    size: 1Gi
+  defaultBuckets: montandon-etl-data,montandon-etl-static  # XXX: Don't change this
+  provisioning:
+    enabled: true
+    resourcesPreset: "nano"
+    cleanupAfterFinished:
+      enabled: true
+    extraCommands:
+      - "mc anonymous set download provisioning/montandon-etl-static"
+
+postgresql:
+  enabled: false
```

Comment:
shouldn't we enable  it?

Reply:
We need this for alpha instances but for prod we will use azure resources